[2024-10-07T13:23:20.792+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-07T13:23:20.800+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: csv_to_star_schema_to_s3_to_redshift.create_dim_customer_table manual__2024-10-07T13:23:12.217197+00:00 [queued]>
[2024-10-07T13:23:20.804+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: csv_to_star_schema_to_s3_to_redshift.create_dim_customer_table manual__2024-10-07T13:23:12.217197+00:00 [queued]>
[2024-10-07T13:23:20.804+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-10-07T13:23:20.813+0000] {taskinstance.py:2888} INFO - Executing <Task(SQLExecuteQueryOperator): create_dim_customer_table> on 2024-10-07 13:23:12.217197+00:00
[2024-10-07T13:23:20.818+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=11570) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-10-07T13:23:20.820+0000] {standard_task_runner.py:72} INFO - Started process 11575 to run task
[2024-10-07T13:23:20.819+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'csv_to_star_schema_to_s3_to_redshift', 'create_dim_customer_table', 'manual__2024-10-07T13:23:12.217197+00:00', '--job-id', '142', '--raw', '--subdir', 'DAGS_FOLDER/sales_pipeline.py', '--cfg-path', '/tmp/tmpl7k_lvay']
[2024-10-07T13:23:20.821+0000] {standard_task_runner.py:105} INFO - Job 142: Subtask create_dim_customer_table
[2024-10-07T13:23:20.833+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-10-07T13:23:20.853+0000] {task_command.py:467} INFO - Running <TaskInstance: csv_to_star_schema_to_s3_to_redshift.create_dim_customer_table manual__2024-10-07T13:23:12.217197+00:00 [running]> on host 66045655c750
[2024-10-07T13:23:20.906+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='csv_to_star_schema_to_s3_to_redshift' AIRFLOW_CTX_TASK_ID='create_dim_customer_table' AIRFLOW_CTX_EXECUTION_DATE='2024-10-07T13:23:12.217197+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-07T13:23:12.217197+00:00'
[2024-10-07T13:23:20.907+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-07T13:23:20.916+0000] {sql.py:266} INFO - Executing: 
        CREATE TABLE IF NOT EXISTS etl_data.dim_customer (
            Customer_Id INT, Gender VARCHAR(255), Customer_Login_type VARCHAR(255), customer_key INT
        );
        
[2024-10-07T13:23:20.923+0000] {base.py:84} INFO - Retrieving connection 'redshift_default'
[2024-10-07T13:23:20.928+0000] {base.py:84} INFO - Retrieving connection 'redshift_default'
[2024-10-07T13:23:20.933+0000] {base.py:84} INFO - Retrieving connection 'aws_default'
[2024-10-07T13:23:20.934+0000] {connection_wrapper.py:388} INFO - AWS Connection (conn_id='aws_default', conn_type='aws') credentials retrieved from login and password.
[2024-10-07T13:23:22.138+0000] {credentials.py:1147} INFO - Found credentials in environment variables.
[2024-10-07T13:23:23.918+0000] {sql.py:509} INFO - Running statement: 
        CREATE TABLE IF NOT EXISTS etl_data.dim_customer (
            Customer_Id INT, Gender VARCHAR(255), Customer_Login_type VARCHAR(255), customer_key INT
        );
        , parameters: None
[2024-10-07T13:23:25.758+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-07T13:23:25.758+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=csv_to_star_schema_to_s3_to_redshift, task_id=create_dim_customer_table, run_id=manual__2024-10-07T13:23:12.217197+00:00, execution_date=20241007T132312, start_date=20241007T132320, end_date=20241007T132325
[2024-10-07T13:23:25.811+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-07T13:23:25.823+0000] {standard_task_runner.py:217} INFO - Process not found (most likely exited), stop collecting metrics
[2024-10-07T13:23:25.825+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-10-07T13:23:25.830+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
